src/textbooks/Textbooks.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Breadcrumb,
+  Button,
+  Container,
+  Layout,
+  Row,
+} from '@edx/paragon';
+import { Add as AddIcon } from '@edx/paragon/icons';
+
+import { useModel } from '../generic/model-store';
+import { LoadingSpinner } from '../generic/Loading';
+import SubHeader from '../generic/sub-header/SubHeader';
+import getPageHeadTitle from '../generic/utils';
+import EmptyPlaceholder from './empty-placeholder/EmptyPlaceholder';
+import TextbookCard from './textbook-card/TextbooksCard';
+import { useTextbooks } from './hooks';
+import messages from './messages';
+
+const Textbooks = ({ courseId }) => {
+  const intl = useIntl();
+
+  const courseDetails = useModel('courseDetails', courseId);
+  document.title = getPageHeadTitle(courseDetails?.name, intl.formatMessage(messages.headingTitle));
+
+  const { textbooks, isLoading, breadcrumbs } = useTextbooks(courseId);
+
+  if (isLoading) {
+    return (
+      <Row className="m-0 mt-4 justify-content-center">
+        <LoadingSpinner />
+      </Row>
+    );
+  }
+
+  return (
+    <Container size="xl" className="px-4">
+      <section className="mb-4 mt-5">
+        <SubHeader
+          title={intl.formatMessage(messages.headingTitle)}
+          breadcrumbs={<Breadcrumb ariaLabel="Textbook breadcrumb" links={breadcrumbs} />}
```

Comment:
[nit]: Let's add a translation for this aria-label

Reply:
Fixed